README.md

```diff
@@ -76,15 +73,21 @@ class MyModelViewset(VisibilityViewMixin, ModelViewSet):
 Data leaks in REST Framework may happen if you use includes (or even only references) from a model the user may see to something that should be hidden.
 To avoid such leaks, make sure to use a subclassed related field (either by creating your own using the provided `VisibilityRelatedFieldMixin`, or by using one of the provided types. See example below).
 
+Similar problems can occur when PATCHing existing relationships where some are hidden through the visibilities.
+The hidden relations will be overwritten by the latest update.
+To prevent that include the `VisibilitySerializerMixin` in your serializer where you use `ManyToMany` relations.
+Preferably after the `ValidatorMixin` so that validations first run and only afterwards it considers PATCHing the relation.
```

Comment:
I'd rephrase this a bit. What exactly happens? Are the hidden relations simply dropped? Also I'd formulate it in a way that "if you use the `VisibilityRelatedFieldMixin`, you will need to use the  `VisibilitySerializerMixin`..."

Also drop the "preferably" - I think this is a *must*

Reply:
Done